### PR TITLE
Spanish Ask style fixes

### PR DESCRIPTION
**Pull request by mistergone**
This Pull Request updates several styles in Spanish-language Ask CFPB to match our latest designs as well as design elements from English-language Ask CFPB.

Additionally, the `gulp watch` task was updated to watch directories matching
`cfgov/legacy/static/*/less/`

## Changes
- Multiple LESS changes to `es-ask-styles.less`
- `gulp watch` directories expanded

## Screenshots
screen shot 2017-12-19 at 8 23 15 am
screen shot 2017-12-19 at 8 24 03 am

## Checklist
* [x] PR has an informative and human-readable title
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [x] Passes all existing automated tests
* [x] Any *change* in functionality is tested
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [x] Project documentation has been updated
* [x] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

**Comment by willbarton** [comment]
There's a little bit of weirdness now around the page link style. 

image

And when the ASK_SEARCH_TYPOS feature flag is turned off, the spacing around the grey bar goes wonky on the landing page:

image

**Comment by mistergone** [comment]
Pagination fixed (?):
screen shot 2017-12-19 at 8 54 23 am

**Comment by willbarton** [comment]
@mistergone ish? should the font size be the same for the current page as the other pages?

**Comment by mistergone** [comment]
I also fixed the other issue @willbarton pointed out:

screen shot 2017-12-19 at 9 37 11 am
screen shot 2017-12-19 at 9 37 16 am

**Comment by nataliafitzgerald** [comment]
@mistergone 

## Pagination
- Page numbers should be Black as the default and Pacific on hover
- Space between the two rows of numbers should be tighter (to match the live site)

| Live site | Build | 
| --- | --- | 
| desktop|mobile

File: cfgov/legacy/static/knowledgebase/less/es-ask-base.less [hunk 1]
```diff
@@ -1,3 +1,137 @@
...
+        margin: 0 0 0.68em;
+        max-width: 670px;
+    }
+}
+
+.full-bleed.content {
+    margin-top: 0em;
+}
+
+.es-ask__results-section {
```
Comment (anselmbradford) [inline cfgov/legacy/static/knowledgebase/less/es-ask-base.less]: Is this correct BEM? (Block-Element__Modifier) So `__results-section` is a modifier on the `es-ask` block. Should this actually be `es-ask_results-section`?

Comment (mistergone) [reply cfgov/legacy/static/knowledgebase/less/es-ask-base.less]: Ooos, good catch, @anselmbradford

**Comment by nataliafitzgerald** [comment]
@mistergone 

## Q & A Labels (P & R)
- These should be the same size. In the build version the "P" is larger than the "R" and indented less. 

| Live site | Build | 
| --- | --- | 
| desktop|mobile

**Comment by mistergone** [comment]
@willbarton - I've just finished an extensive style review with @nataliafitzgerald who has approved the changes! Sorry for the bugs and delay, but this is ready to go now!

**Pull request merged**